infra/dev/nginx/nginx.conf

```diff
@@ -0,0 +1,37 @@
+http {
```

Comment:
Why is this needed? When I did my simple version, I just had `server { ...`

Reply:
We need an empty `events` sections to a parent `server` block is therefore mandatory.